meltano.yml

```diff
@@ -19,6 +19,8 @@ plugins:
       kind: password
     - name: additional_auth_tokens
       kind: array
+    - name: auth_app_keys
```

Comment:
For simplicity, I'm opting to only create an `auth_app_keys` option and not mirror the `auth_token` + `additional_auth_tokens` structure.

Reply:
Can we add this to https://github.com/MeltanoLabs/tap-github/blob/53057a2388bb70c0919ce53c017106f8bdb174f7/tap_github/tap.py#L36 as well?

There's some unfortunate amount of duplication between that and `meltano.yml` at the moment.

Reply:
Will update later tonight or tomorrow, thanks for catching.  I'll add `expiry_time_buffer` too while I'm add it, I missed adding an entry for it in https://github.com/MeltanoLabs/tap-github/pull/289.

Reply:
Thanks!